pkg/deb/postinstall

```diff
@@ -1,6 +1,8 @@
 
 mkdir -p /var/log/serviced
-chmod 1777 /var/log/serviced
+chgrp serviced /var/log/serviced
```

Comment:
IDK. The problem we're trying to solve is that users logged into a CC node, who are not in the `serviced` group, should not be able to execute the CC CLI or look at CC config data.

So far, the containerized services seem to have been running fine after this change with the exception of ZEN-26903 which is under investigation